test/bugs-reported/bug4.future

```diff
@@ -0,0 +1,28 @@
+bug: The List Module does not support accessing elements of a list based on index. I feel that this should be supported.
```

Comment:
I have split bug4 into 2 bugs: bug8.chpl and bug9.chpl with respective future, good and bad files.

I agree that .these() iterator should not be used for accessing list elements by index. But, it should throw an error like "Iterator does not allow indexed access."